apps/meteor/tests/end-to-end/apps/video-conferences.ts

```diff
@@ -284,64 +335,224 @@ describe('Apps - Video Conferences', () => {
 		});
 
 		describe('[/video-conference.info]', () => {
-			let callId: string | undefined;
+			describe('[basic provider]', () => {
+				let callId: string | undefined;
+
+				before(async () => {
+					await updateSetting('VideoConf_Default_Provider', 'test');
+					const res = await request.post(api('video-conference.start')).set(credentials).send({
+						roomId,
+					});
+
+					callId = res.body.data.callId;
+				});
+
+				it('should load the video conference data successfully', async () => {
```

Comment:
Shouldn't we include a test case with the basic provider AND the persistent chat setting enabled?